combo/xiaoyu.yaml

```diff
@@ -58,10 +59,32 @@
 - combo: "RDS qcf+K>K, RDS cancel, cr.MK xx qcb+MP, qcb+LP, qcf+LP/cr.HK"
   damage: 384/375
   tags: [yolo, 0bar, medium, bnb]
+- combo: "st.MK, cr.LP xx qcf+HP, RDS P>P>d, AOP P, st.MP"
+  damage: 377/374
+  comment: "Enders: `ender /^AOP P, st.MP`"
+  tags: [yolo, 0bar, hard, hitconfirm]
 
 # juggle
 - combo: "cl.HP xx qcb+LP, qcb+LP, qcf+LP/cr.HK"
   tags: [juggle, 0bar, easy, bnb]
+- combo: "qcb+LP, cl.HP xx qcb+LP, qcf+LP/cr.HK"
+  tags: [juggle, 0bar, easy, bnb]
 - combo: "cr.HP xx CADC, cl.HP xx d+PPP, AOP cancel, cr.MK xx qcb+MP,
   qcf+LP/cr.HK"
-  tags: [juggle, 0bar, hard]
+tags: [juggle, 0bar, hard]
+
+#punish
+- combo: "qcf+KK, cr.HP xx qcb+LP, qcf+LP/cr.HK"
+  damage: 343/333
+  tags: [punish, 1bar, easy]
+
+# enders
+- combo: "AOP P, st.MP xx qcb+MP, qcb+LP, cr.HK"
+  comment: "Starters: `/AOP P, st.MP$`"
+  tags: [ender, 0bar, easy]
+- combo: "AOP P, st.MP xx qcb+MP, qcb+LP, qcf+LP"
+  comment: "Starters: `/AOP P, st.MP$`"
+  tags: [ender, 0bar, easy]
+- combo: "AOP P, st.MP xx d+PPP AOP qcf+K MP+MK (first hit)"
+  comment: "Starters: `/AOP P, st.MP$`"
+  tags: [ender, 1bar, medium]
```

Comment:
Including the switch cancel is something new, but I'm okay with adding it. I just prefer the notation to be `qcf+K(1) MP+MK`. We might as well start a new tag for it as well (`switch`?). Also I usually put a comma after `d+PPP` because technically you do recover between that command and the next.